catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -36,43 +39,69 @@
 import org.codice.ddf.registry.common.metacard.RegistryObjectMetacardType;
 import org.codice.ddf.spatial.ogc.csw.catalog.common.CswSourceConfiguration;
 import org.codice.ddf.spatial.ogc.csw.catalog.common.source.AbstractCswStore;
+import org.geotools.filter.FilterFactoryImpl;
+import org.opengis.filter.Filter;
+import org.opengis.filter.FilterFactory;
 import org.osgi.framework.BundleContext;
+import org.osgi.service.cm.Configuration;
+import org.osgi.service.cm.ConfigurationAdmin;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.google.common.base.Charsets;
 import com.thoughtworks.xstream.converters.Converter;
 
 import ddf.catalog.Constants;
 import ddf.catalog.data.Attribute;
 import ddf.catalog.data.Metacard;
+import ddf.catalog.data.Result;
 import ddf.catalog.data.impl.AttributeImpl;
 import ddf.catalog.filter.delegate.TagsFilterDelegate;
 import ddf.catalog.operation.OperationTransaction;
+import ddf.catalog.operation.Query;
 import ddf.catalog.operation.QueryRequest;
 import ddf.catalog.operation.SourceResponse;
 import ddf.catalog.operation.UpdateRequest;
 import ddf.catalog.operation.UpdateResponse;
+import ddf.catalog.operation.impl.QueryImpl;
+import ddf.catalog.operation.impl.QueryRequestImpl;
 import ddf.catalog.operation.impl.SourceResponseImpl;
 import ddf.catalog.source.IngestException;
+import ddf.catalog.source.SourceMonitor;
 import ddf.catalog.source.UnsupportedQueryException;
 import oasis.names.tc.ebxml_regrep.xsd.rim._3.ExternalIdentifierType;
 import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryObjectType;
 
 public class RegistryStoreImpl extends AbstractCswStore implements RegistryStore {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(RegistryStoreImpl.class);
+
     public static final String PUSH_ALLOWED_PROPERTY = "pushAllowed";
 
     public static final String PULL_ALLOWED_PROPERTY = "pullAllowed";
 
+    public static final String REMOTE_NAME = "remoteName";
+
+    public static final String REGISTRY_ID = "registryId";
```

Comment:
Is there a reason we can't just use the registry-id constant we have in the metacard type?